src/main/java/com/conveyal/analysis/controllers/OpportunityDatasetController.java

```diff
@@ -148,6 +148,8 @@ private OpportunityDatasetUploadStatus downloadLODES(Request req, Response res)
         }
 
         final String regionId = req.params("regionId");
+        final int zoom = req.params("zoom") != null ? Integer.parseInt(req.params("zoom")) : DEFAULT_ZOOM;
```

Comment:
Here we probably don't want to use `req.params` either, otherwise it would have to be part of the URL. We could put as a query parameter fairly easily. So `req.queryParams("zoom")` would work here.